src/filesinkhelper.ipp

```diff
@@ -85,9 +85,9 @@ namespace g3 {
          return ss_entry.str();
       }
 
-      std::string createLogFileName(const std::string &verified_prefix) {
+      std::string createLogFileName(const std::string &verified_prefix, const std::string &logger_id=".g3log.") {
          std::stringstream oss_name;
-         oss_name << verified_prefix << ".g3log.";
+         oss_name << verified_prefix << logger_id;
```

Comment:
To get it going, I had to unzip the gtest zip file manually.

I would suggest to use something like the following in your CMake logic,
so that you just pull it down and build instead of having the zip file in your repo.
This is much easier I think and it works automatic.

This is from some lib i m using so it might require a little tweaking but it should be pretty straightforward.
```
include(ExternalProject)

set( GTEST_SOURCE_DIR ${EP_BASE}/Source/gtest )
set( GTEST_BINARY_DIR ${EP_BASE}/Build/gtest )
  
#externalproject_add(gtest-download
externalproject_add(gtest
  URL              https://googletest.googlecode.com/files/gtest-1.7.0.zip
  URL_HASH         SHA1=f85f6d2481e2c6c4a18539e391aa4ea8ab0394af
  SOURCE_DIR       ${GTEST_SOURCE_DIR}
  BINARY_DIR       ${GTEST_BINARY_DIR}
  CMAKE_ARGS       ${gtest_cmake_args}
                   -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS}
                   -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
  INSTALL_COMMAND  ""
  )

set( GTEST_INCLUDE_DIRS ${GTEST_SOURCE_DIR}/include )
if(BUILD_SHARED_LIBS)
  set( GTEST_LIBRARY ${GTEST_BINARY_DIR}/libgtest${CMAKE_SHARED_LIBRARY_SUFFIX} )
else()
  set( GTEST_LIBRARY ${GTEST_BINARY_DIR}/libgtest${CMAKE_STATIC_LIBRARY_SUFFIX} )
endif()
  
add_custom_command(
  #TARGET gtest-download
  TARGET gtest
  POST_BUILD
  COMMAND ${CMAKE_COMMAND} -E copy_if_different ${GTEST_LIBRARY} lib
  COMMENT "Copying file to Runtime directory: " $<TARGET_FILE:${GTEST_LIBRARY}>
  )

install( FILES ${GTEST_LIBRARY} DESTINATION lib/${CMAKE_PROJECT_NAME}/3rd )
```